src/lib-components/fileUploader.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+import {sectionHeader} from "@/base/helpers";
+
+export async function globalFileUpload(file) {
```

Comment:
you need to provide a documentation of why does this function exists (provide some context) and how to use this function in the Readme of the library

Reply:
Updated